packages/monosize-storage-azure/src/types.mts

```diff
@@ -1,3 +1,16 @@
+/**
+ * @see https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-azure-hosted-applications
+ */
+export type AzureAuthenticationType = 'AzureNamedKeyCredential' | 'AzurePipelinesCredential';
+
 export type AzureStorageConfig = {
   endpoint: string;
+  /**
+   * @default 'AzureNamedKeyCredential' auth type
+   */
+  authType?: AzureAuthenticationType;
+  /**
+   * @default 'latest' table name
+   */
+  tableName?: string;
```

Comment:
Added `tableName` here to allow for passing custom table names. This also allows users to re-use the same azure storage and function app resources (i.e. instead of having to re-create a new azure storage and azure function app for each fluent related repo, those resources can simply be shared just with each repo having their own `tableName` configured)

Reply:
this is awesome addition, ty!